dag_status.py

```diff
@@ -0,0 +1,38 @@
+import time
+import requests
+
+def get_latest_state():
+    url = "http://localhost:8080/api/v1/dags/lakeFS_workflow/dagRuns"
+    username="admin"
+    password="admin"
+    response = requests.get( url,  auth=(username, password))
+    dag_details = {}
+    for key in response.json()['dag_runs']:
+        dag_details[key['execution_date']] = key['state']
+
+    # Creates a sorted dictionary (sorted by key)
+    from collections import OrderedDict
+
+    dict1 = OrderedDict(sorted(dag_details.items(), reverse=True))
+    state = dict1[list(dict1.keys())[0]]
+    return state
+
+def dag_state():
+    print("Inside the dag state block")
```

Comment:
This looks like a leftover debug print.  Can we remove it now?

Reply:
removed.